edalize/edatool.py

```diff
@@ -309,10 +309,18 @@ def _run_tool(self, cmd, args=[]):
         logger.debug("Running " + cmd)
         logger.debug("args  : " + ' '.join(args))
 
+        # This looks a little odd, but it is necessary in case the environment
+        # sometimes gets updated after the backend is instantiated. Making the
+        # handling of environments consistent across backends could resolve
+        # this.
+        env = self.env
```

Comment:
Was this intended to be:
```suggestion
        env = self.env.copy()
```

Reply:
Yes, you are right. I'm going to rewrite my comment on this to be clearer though - it took me a while to remember what I was talking about last month. :)